mega_analysis/semiology.py

```diff
@@ -121,7 +122,7 @@ def remove_exclusions(
             include_cortical_stimulation: bool,
             include_et_topology_ez: bool,
             include_spontaneous_semiology: bool,
-            ) -> pd.DataFrame:
+    ) -> pd.DataFrame:
```

Comment:
Please stop de-indentating these brackets.

Reply:
this is automatic pep8 and linting which I enabled because of issue #50

Reply:
I know, but this is not explicitly specified in PEP8 and you can just not stage these changes for the commit. The point is to make the code more readable, and this doesn't as indentation is lost.